README.md

```diff
@@ -10,5 +10,66 @@ make install
 make start
 ```
 
+## Requirements
+
+Application requires few other services and configuration in environment
+variables. Docker-compose in this repository is configured to read variables
```

Comment:
The API requires other services (keycloak, database, etc) in order to function.  Some of the configuration comes from environment variables.  Docker-compose in this........